src/components/Widget/components/Conversation/components/Messages/styles.scss

```diff
@@ -78,12 +85,24 @@
 
   .messages-container {
     @include messages-container-fs;
+
+    .avatar {
+      width: 2em;
+      height: 2em;
+      margin: 0.5em 0 0 -1em;
+      padding: 0;
+    }
   }
 }
 
 @media screen and (max-width: 800px)  {
   .messages-container {
     @include messages-container-fs;
+    padding: 3% 4% 0 4%!important;
```

Comment:
why using `!important?`

Reply:
we use !important to guarantee that this param will not be overwrite by the padding on @include class